openIMIS/openIMIS/settings.py

```diff
@@ -158,7 +165,8 @@ def SITE_URL():
     "django_apscheduler",
     "channels",  # Websocket support
     "developer_tools",
-    "drf_spectacular"  # Swagger UI for FHIR API
+    "drf_spectacular",  # Swagger UI for FHIR API
+    'django_nose'
```

Comment:
Please use double quotation to keep it consistent with other strings in this file.